Dockerfile

```diff
@@ -0,0 +1,34 @@
+#ARG INSTALL_PYTHON_VERSION=${INSTALL_PYTHON_VERSION:-3.8.12}
+#FROM python:${INSTALL_PYTHON_VERSION}-slim-bullseye AS base
+# FROM debian:bullseye-slim AS base
+FROM ssthapa/django5:0.1
```

Comment:
Where is this coming from?

Reply:
That's something I built.  The source is at https://github.com/sthapa/django-images